voluspa.py

```diff
@@ -92,6 +93,8 @@ async def on_ready():
 
     bot.loop.create_task(update_status_task(bot, quotes))
 
+    await bot.tree.sync()
```

Comment:
What is this doing?

Reply:
This syncs the application commands (slash commands) with discord at startup

[API Reference](https://discordpy.readthedocs.io/en/stable/interactions/api.html#discord.app_commands.CommandTree.sync)
[Example](https://github.com/Rapptz/discord.py/blob/master/examples/app_commands/basic.py#L28)

Reply:
I.e. that's how you get the pretty discord context menu when you type /. Until now the paradigm has been Voluspa listens for commands in messages, now the command invocation is handled by Discord. That's my understanding.